pkg/controller/podgc/gc_controller.go

```diff
@@ -77,15 +77,16 @@ func NewPodGC(ctx context.Context, kubeClient clientset.Interface, podInformer c
 		nodeLister:             nodeInformer.Lister(),
 		nodeListerSynced:       nodeInformer.Informer().HasSynced,
 		nodeQueue:              workqueue.NewNamedDelayingQueue("orphaned_pods_nodes"),
-		deletePod: func(namespace, name string) error {
-			klog.InfoS("PodGC is force deleting Pod", "pod", klog.KRef(namespace, name))
-			return kubeClient.CoreV1().Pods(namespace).Delete(ctx, name, *metav1.NewDeleteOptions(0))
-		},
 	}
 
 	return gcc
 }
 
+func (gcc *PodGCController) deletePod(ctx context.Context, namespace, name string) error {
```

Comment:
nit: move this after all the exported methods

Reply:
moved